react/components/atoms/avatar/avatar.stories.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { withKnobs } from "@storybook/addon-knobs";
+
+import { Avatar } from "../../";
+
+storiesOf("Atoms", module)
+    .addDecorator(withKnobs)
+    .add("Avatar", () => {
+        return (
+            <Avatar
+                image={{ uri: "https://semantic-ui.com/images/avatar2/large/matthew.png" }}

```

Comment:
Why not `https://id.platforme.com/admin/accounts/v-fl%40platforme.com/avatar`? :D (change it in https://id.platforme.com/)